code/items/weapons/TTTWeapon.cs

```diff
@@ -7,17 +7,6 @@
 
 namespace TTTReborn.Items
 {
-    // [AttributeUsage(AttributeTargets.Class, Inherited = false)]
-    // public class WeaponAttribute : LibraryAttribute
-    // {
-    //     public WeaponType WeaponType;
-
-    //     public WeaponAttribute(string name) : base(name)
-    //     {
-
-    //     }
-    // }
-
```

Comment:
Please keep it as this should be added by sbox soonTM (so we gonna be able to fetch data without the need initializing and spawning such a weapon),

Reply:
I'm a simple man - I see comment, I delete it 🤣 
I'll revert it and add additional clarification